Calculator.Tests/ExpressionParserTests.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Calculator.Core.Extensions;
+using Calculator.Infrastructure.Data;
+
+namespace Calculator.Tests
+{
+	public class ExpressionParserTests
+	{
+		private readonly ExpressionParser parser = CreateExpressionParser();
+
+		[Fact]
+		public void ParseAndEvaluate_ReturnsSameValuesIfArithmeticExpressionWithPositiveValuesIsPassed()
+		{
+			var expression = "2 + 1 - 3";
```

Comment:
you can use InlineData attribute to pass expression and excpected result through it. check it in other tests